solution/backend/regulations/views/regulation_landing.py

```diff
@@ -24,6 +24,7 @@ def get_context_data(self, **kwargs):
 
         parts = Part.objects.effective(date.today()).filter(title=title)
         reg_version = current.date.isoformat()
+        reg_version_string = datetime.strftime(current.date, "%b %-d, %Y")
```

Comment:
Why format this one in the context and not in the view like last_updated?  Both are fine, just curious why the difference.